src/vmm/src/builder.rs

```diff
@@ -168,7 +168,9 @@ pub mod aarch64 {
         if cmdline_contains_console {
             // Make stdout non-blocking.
             set_stdout_nonblocking();
-            let serial = setup_serial_device(event_manager, std::io::stdin(), std::io::stdout())?;
```

Comment:
The last "fix: Refactor vmm builder code to simplify logic" should be squashed with previous one.